app/src/main/java/org/mozilla/fenix/library/bookmarks/BookmarkFragment.kt

```diff
@@ -365,6 +365,9 @@ class BookmarkFragment : Fragment(), CoroutineScope, BackHandler, AccountObserve
         getManagedEmitter<SignInChange>().onNext(SignInChange.SignedOut)
     }
 
+    override fun onAuthenticationProblems() {
```

Comment:
Let's leave this for later, but having many accountobserver implementations doesn't feel right.